docs: Document MCP project/folder toggle (#4819)
n8n-io/n8n-docs: docs/advanced-ai/mcp/accessing-n8n-mcp-server.md

2. Open the menu on a workflow card.
3. Select **Enable MCP access**.

### Enabling access for projects/folders

/// info | Available from n8n v2.24.0
///

> You can use breadcrumbs menu in workflows list to toggle MCP access for all workflows in the current project or a folder:

Reviewer: ```suggestion
You can use the <span class="n8n-inline-image">![Options menu](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> in the workflow list to toggle MCP access for all workflows in the current project or folder:
```

> 1. Navigate to the desired project/folder workflow list

Reviewer: ```suggestion
1. Navigate to the desired project and select **Workflows** from the top menu, then open a subfolder if required.
```

> You can use the <span class="n8n-inline-image">![Options menu](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> in the workflow list to toggle MCP access for all workflows in the current project or folder:

Reviewer: ```suggestion
You can use the **Options** menu <span class="n8n-inline-image">![Options menu](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> in the workflow list to toggle MCP access for all workflows in the current project or folder:
```

> 2. Click the `...` menu next to the project/folder name

Reviewer: ```suggestion
2. Click the <span class="n8n-inline-image">![Options icon](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> next to the name of the project or folder.
```

> 3. Select an option from `Manage MCP access` menu

Reviewer: ```suggestion
3. Select **Manage MCP access**, then either **Enable MCP** or **Disable MCP**.
```

> 2. Click the <span class="n8n-inline-image">![Options icon](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> next to the name of the project or folder.

Reviewer: ```suggestion
2. Select the **Options** menu <span class="n8n-inline-image">![Options icon](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> next to the name of the project or folder.
```
